festim/mesh/mesh_1d.py

```diff
@@ -11,10 +11,10 @@ class Mesh1D(F.Mesh):
     1D Mesh
 
     Args:
-        vertices (list): the mesh x-coordinates (m)
+        vertices (list or np.array): the mesh x-coordinates (m)
 
     Attributes:
-        vertices (list): the mesh x-coordinates (m)
+        vertices (np.array): the mesh x-coordinates (m)
```

Comment:
```suggestion
        vertices (np.ndarray): the mesh x-coordinates (m)
```

Reply:
@JonathanGSDUFOUR it looks like this slipped! 😄

Reply:
Oh yeah sorry, not used to git 😅  yet 😉